verify/index.ts

```diff
@@ -51,23 +47,25 @@ function configTrustStore(dir: string) {
             let trustStore = trustStores[j]; // .github/truststore/x509/ca/<my_store>
             let trustStoreName = path.basename(trustStore); // <my_store>
             let certFile = getFileFromDir(trustStore); // [.github/truststore/x509/ca/<my_store>/<my_cert1>, .github/truststore/x509/ca/<my_store>/<my_cert2>, ...]
-            execSync(`notation cert add -t ${trustStoreType} -s ${trustStoreName} ${certFile.join(' ')}`, { encoding: 'utf-8' });
+            for (const cert of certFile) {
+                await exec.getExecOutput(`notation cert add -t ${trustStoreType} -s ${trustStoreName} "${cert}"`);
+            }
```

Comment:
I think you can try something like
```ts
exec.getExecOutput('notation', ['cert', 'add', '-t', trustStoreType, '-s', trustStoreName, 'cert1.crt', 'hello world.crt']);
```

Basically, an array of arguments.

Reply:
Updated to `exec.getExecOutput(`notation cert add -t ${trustStoreType} -s ${trustStoreName}`, certFile);`
Because `certFile` is the args array for this command.